src/test/java/io/jenkins/plugins/opentelemetry/init/GitHubClientMonitoringTest.java

```diff
@@ -5,10 +5,14 @@
 
 package io.jenkins.plugins.opentelemetry.init;
 
-import junit.framework.TestCase;
+import org.junit.Rule;
 import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
 
-public class GitHubClientMonitoringTest extends TestCase {
+public class GitHubClientMonitoringTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
```

Comment:
When run against recent versions of the GitHub branch source plugin, this test was failing because those recent versions assume that Jenkins is running. Fixed by starting Jenkins during the test.